constellations/gemini-admin/src/location/location.sass

```diff
@@ -1,26 +1,26 @@
-@import "../static.styl"
+@import "../_.constants.sass"
 
 #view-location
     position: relative
     width: 100%
     height: 400px
-    padding: 0 spacing2x
+    padding: 0 $spacing2x
     box-sizing: border-box
-    color: colorBlackDark
+    color: $colorBlackDark
 
     & h1
```

Comment:
you know what, let's make it consistent.

Anytime you see:

```
& h1
    ...
```

Just replace the `& h1` with `h1`. With the ampersand.
& selects the parent which is redundant if it's already indented